lib/ansible/modules/cloud/vmware/vmware_guest_register_operation.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, sky-joker
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+module: vmware_guest_register_operation
+short_description: VM inventory registration operation
+author:
+  - sky-joker (@sky-joker)
+version_added: '2.9'
+description:
+  - This module can register or unregister VMs to the inventory.
+requirements:
+  - python >= 2.6
+  - PyVmomi
+options:
+  datacenter:
+    description:
+    - Destination datacenter for the register/unregister operation.
+    - This parameter is case sensitive.
+    default: ha-datacenter
+  cluster_name:
```

Comment:
These are random comments from someone extremely wanting to see this module (so much that I've pulled it and am testing it out).  Other vmware modules use cluster instead of cluster_name.  Would be nice if it were consistent throughout all the modules.

Reply:
Thank you for comment!
Modify cluster_name to cluster.

Reply:
Modified cluster_name parameter name to cluster. :)